central/telemetry/marketing/interceptor.go

```diff
@@ -0,0 +1,54 @@
+package marketing
```

Comment:
My understanding is that this file is for instrumenting API calls to gather related telemetry. Let's make it clear from naming. A couple of suggestions:
* introduce a named function that returns an interceptor `func` based on passed `telemeter` and `trackedPaths`. You can then call it something like `TelemetryAPIInterceptor(...) grpc.UnaryServerInterceptor {...}`
* call the file `api_interceptor.go`
* or move it to `telemetry/interceptors/api.go`